incubator/group/keeper.go

```diff
@@ -523,8 +539,10 @@ func (k Keeper) CreateProposal(ctx sdk.Context, accountAddress sdk.AccAddress, c
 	if err != nil {
 		return 0, errors.Wrap(err, "end time conversion")
 	}
-
-	// todo: prevent impossible case case with threshold >= group total
+	// prevent proposal that can not succeed
+	if policy.GetThreshold() != nil && policy.GetThreshold().Threshold.GTE(g.TotalWeight) {
+		return 0, errors.Wrap(ErrInvalid, "total group weight not greater than policy threshold")
```

Comment:
This msg is bit confusing. Should it be `policy threshold should not be greater than the total group weight`?